fluidBuild.config.cjs

```diff
@@ -60,7 +60,7 @@ module.exports = {
 		},
 		"api-extractor:commonjs": ["tsc"],
 		"api-extractor:esnext": {
-			dependsOn: ["build:esnext"],
+			dependsOn: ["api-extractor:commonjs", "build:esnext"],
```

Comment:
Please add a comment for the dependency - it is not obvious why it would be needed.

Reply:
I removed this again. If in the future we have some verification that runs in one of the tasks, we can introduce this dependency.

Reply:
Seems like `api-extractor-base-esm.json` should have `apiReport: { enabled: true }` and then we see that commonjs and ems are the same. Can be later.